Makefile

```diff
@@ -3,14 +3,34 @@ clean:
 	rm -f dist/*.whl
 	rm -rf build
 
-build:
+# PyPI archive preparation
+build_pypi:
 	python setup.py sdist
+	python -m cython -a streaming_form_data/_parser.pyx -o annotation.html
```

Comment:
Please move this command to a `annotate` task, and introduce that as a dependency for the `build` task.

```
annotate:
  cython -a streaming_form_data/_parser.pyx -o annotation.html

build: annotate
  python setup.py sdist
```

Reply:
good point

I would also say it could be nice to specify dependencies and output files here. So make could skip steps faster.

I will read documentation about it. I never used make much.